### PR TITLE
Migrate OpenGL string-literals to embedded files

### PR DESCRIPTION
**Pull request by Wunkolo**
Rather than declaring a big C++ literal for these strings, they are now
proper `.vert` and `.frag` files that will become embedded into the
application at build-time. It also allows for clang-format to format the
glsl files the same as our C++ code. CMake will also automatically track
to re-embed the file if it detects that the glsl source files have
changed since the last build. Ex, making a change to `opengl_display.frag` and compiling will
automatically recompile and link the resource-target that it is
associated with.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -185,6 +187,16 @@ if(ENABLE_OPENGL)
 
     set(HEADER_FILES ${HEADER_FILES} ${RENDERER_GL_INCLUDE_FILES})
     source_group("Source Files\\Core\\OpenGL Renderer" FILES ${RENDERER_GL_SOURCE_FILES})
+
+	cmrc_add_resource_library(
+		resources_renderer_gl
+		NAMESPACE RendererGL
+		WHENCE "src/host_shaders/"
+		"src/host_shaders/opengl_display.frag"
```
Comment (wheremyfoodat) [inline CMakeLists.txt:195]: "WHENCE"
image

**Comment by wheremyfoodat** [comment]
(The draft tag is intentional right?)

**Comment by Wunkolo** [comment]
> (The draft tag is intentional right?)

Yea I wanted to test this on some more platforms locally before I have it ready to merge.

**Comment by Wunkolo** [comment]
Ready for approval 👍

**Comment by wheremyfoodat** [comment]
I need to make the glsl files show up in Visual Studio with source_group stuff, gimme a sec I'm looking at the ceiling

**Pull request merged**